docs/front-matter.md

```diff
@@ -0,0 +1,195 @@
+- [Permalink](/front-matter.md#permalink)
+- [Source](/front-matter.md#source)
+- [Concurrence](/front-matter.md#concurrence)
+- [Plain language certification](/front-matter.md#plain-language-certification)
+- [Collections, children, and ordering](/front-matter.md#collections-children-and-ordering)
+- [Major vs. Related links](/front-matter.md#major-vs-related-links)
+
+### Layout
+
+This sets a page up to use breadcrumb navigation. This is the **default setting** for static pages within our core public-facing content and should not need to be changed.
+
+`layout: page-breadcrumbs.html`
+
+### Template
+
+There are 3 standard template options for content:
+
+| Template                   | Use                                                                                                                                                                                                                                                                                                                                                                                      |
+|----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `template: level2-index `  | Used for hub pages like [va.gov/education/index.html](https://www.va.gov/education/). This template can display content and link lists, right-side promo and social contact links, but no left-side accordian navigation.                                                                                                                                                                |
+| `template: detail-page `   | Used for detail pages like [va.gov/education/about-gi-bill-benefits/](https://www.va.gov/education/about-gi-bill-benefits/). Displays the same elements as `topic-landing`, in addition to previous/next page navigation at the bottom of the page to navigate through sequential content. Can also be used for landing pages that would benefit from the previous/next page navigation. |
+| `template: topic-landing ` | Used for deeper landing pages like [va.gov/disability/eligibility/special-claims/](https://www.va.gov/disability/eligibility/special-claims/). This template can display content, side navigation, main links, and related links.                                                                                                                                                        |
+
+### Title vs. Display Title vs Heading
+
+**Title** is the full title displayed at the top of the page. Example:
+
+`title: VA Burial Benefits and Memorial Items`
+
+It is also what Google uses for SEO meta content. 
```

Comment:
Can we add a note here that this will be used as the name of the tab or bookmark?

Reply:
edited to say:
This is also the text that:
* Is used as the name of the tab and bookmark
* Google uses for SEO meta content